airbyte-integrations/connectors/destination-bigquery-denormalized/src/main/java/io/airbyte/integrations/destination/bigquery/formatter/DefaultBigQueryDenormalizedRecordFormatter.java

```diff
@@ -235,10 +237,32 @@ public Schema getBigQuerySchema(final JsonNode jsonSchema) {
     };
   }
 
+  private static JsonNode getFileDefinition(final JsonNode fieldDefinition) {
+    if (fieldDefinition.has(TYPE_FIELD)) {
+      return fieldDefinition;
+    } else {
+      if (fieldDefinition.has(ANY_OF_FIELD) && fieldDefinition.get(ANY_OF_FIELD).isArray()) {
+        var fieldOptional = MoreIterators.toList(fieldDefinition.get(ANY_OF_FIELD).elements()).stream().findFirst();
+        if (fieldOptional.isPresent()) {
+          return getFileDefinition(fieldOptional.get());
+        }
```

Comment:
PLease, try to use .findFirst().ifPresent(DefaultBigQueryDenormalizedRecordFormatter::getFileDefinition) it should work.

Reply:
`ifPresent` return void, but I need to return JsonNode